scripts/xjoin-config/elasticsearch-sink.json

```diff
@@ -0,0 +1,56 @@
+{
+    "name": "xjoin.inventory.es",
+    "config": {
+      "type.name": "_doc",
+      "connector.class": "io.confluent.connect.elasticsearch.ElasticsearchSinkConnector",
+      "behavior.on.null.values": "delete",
+      "errors.log.include.messages": "true",
+      "transforms": "valueToKey, extractKey, expandJSON, expandPRSJSON, deleteIf, flattenList, flattenListString, flattenPRS, renameTopic",
+      "max.retries": "8",
+      "transforms.flattenList.keys": "namespace,key,value",
+      "transforms.extractKey.type": "org.apache.kafka.connect.transforms.ExtractField$Key",
+      "transforms.expandJSON.sourceFields": "tags",
+      "transforms.expandPRSJSON.sourceFields": "per_reporter_staleness",
+      "transforms.flattenPRS.rootKey": "reporter",
+      "transforms.flattenPRS.sourceField": "per_reporter_staleness",
+      "errors.log.enable": "true",
+      "transforms.deleteIf.type": "com.redhat.insights.deleteifsmt.DeleteIf$Value",
+      "transforms.flattenPRS.type": "com.redhat.insights.flattenlistsmt.FlattenList$Value",
+      "transforms.flattenListString.delimiterJoin": "/",
+      "transforms.flattenListString.type": "com.redhat.insights.flattenlistsmt.FlattenList$Value",
+      "max.in.flight.requests": "1",
```

Comment:
The `max.in.flight.requests` can be removed, default `5` should work?

Reply:
Done